File: Diving into JavaScript: For Beginners/step07/7.02/7.2.02.js
/*

Напишите программу, которая запрашивает ввод строки. Если эта строка содержит символ "д" или символ "н", 
то программа выводит всю строку в верхнем регистре, а иначе выводится строка в нижнем регистре.

Sample Input 1:

ода

Sample Output 1:

ОДА

Sample Input 2:

Дом 

Sample Output 2:

дом

Sample Input 3:

Савелий Крамаров

Sample Output 3:

савелий крамаров

Sample Input 4:

Мона Лиза

Sample Output 4:

МОНА ЛИЗА

*/

let str = prompt();

if (str.includes("д") || str.includes("н")) {
    console.log(str.toUpperCase());
} else {
    console.log(str.toLowerCase());
}

File: Good, good Python/step 07/7.02/7.2.02.py
'''

Подвиг 2. Объявите функцию с именем is_triangle, которая принимает три стороны треугольника (целые числа) и проверяет, 
можно ли из переданных аргументов составить треугольник. (Напомню, что у любого треугольника длина третьей стороны всегда должна быть 
меньше суммы двух других). Если  проверка проходит, вернуть булево значение True, иначе - значение False.

Вызывать функцию не нужно, только задать.

Sample Input:

3 4 5

Sample Output:

True

'''

def is_triangle(a,b,c):
    if c < a + b:
        return True
    else:
        return False
